build/pipelines/azure-pipelines.ci.yaml

```diff
@@ -15,46 +15,63 @@ pr:
 
 name: 0.$(Date:yyMM).$(DayOfMonth)$(Rev:rr).0
 
-jobs:
-- template: ./templates/build-single-architecture.yaml
-  parameters:
-    platform: x64
-    isOSSBuild: true
+resources:
+  repositories:
+  - repository: 1esPipelines
+    type: git
+    name: 1ESPipelineTemplates/1ESPipelineTemplates
+    ref: refs/tags/release
 
-- template: ./templates/build-single-architecture.yaml
+extends:
+  template: v1/1ES.Unofficial.PipelineTemplate.yml@1esPipelines
   parameters:
-    platform: x86
-    isOSSBuild: true
-    condition: not(eq(variables['Build.Reason'], 'PullRequest'))
+    pool:
+      name: EssentialExperiencesOpenSource-windows-2022
+      image: MMSWindows2022-Secure
+      os: windows
 
-- template: ./templates/build-single-architecture.yaml
-  parameters:
-    platform: ARM
-    isOSSBuild: true
-    condition: not(eq(variables['Build.Reason'], 'PullRequest'))
+    sdl:
+      binskim:
+        enabled: false
 
-- template: ./templates/build-single-architecture.yaml
-  parameters:
-    platform: ARM64
-    isOSSBuild: true
-    condition: not(eq(variables['Build.Reason'], 'PullRequest'))
+    stages:
+    - stage: Calculator
+      jobs:
+      - template: /build/pipelines/templates/build-single-architecture.yaml@self
+        parameters:
+          platform: x64
 
-- template: ./templates/run-ui-tests.yaml
-  parameters:
-    platform: x64
-    isOSSBuild: true
-    runsettingsFileName: CalculatorUITests.ci.runsettings
+      - template: /build/pipelines/templates/build-single-architecture.yaml@self
+        parameters:
+          platform: x86
+          condition: not(eq(variables['Build.Reason'], 'PullRequest'))
 
-- template: ./templates/run-unit-tests.yaml
-  parameters:
-    platform: x64
-    isOSSBuild: true
+      - template: /build/pipelines/templates/build-single-architecture.yaml@self
+        parameters:
+          platform: ARM
+          isOSSBuild: true
```

Comment:
Remove `isOSSBuild`?

Reply:
Good point.
Since we are adding back `nuget-security-analysis`, now we need to keep these `isOSSBuild`s. Reverted those removed ones.